SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Reporting a Security Issue
+
+To report a security issue, please email ??? with a description of the issue,
+the steps you took to create the issue, affected versions, and, if known,
+mitigations for the issue. The Security Team will attempt to respond within a
+reasonable timeframe to your email. If the issue is confirmed as a
+vulnerability, we will open a Security Advisory.
+
+## Procedure
+
+1. A GitHub Security Advisory will be created in the appropriate repository.
+2. A project member works privately with the reporter to resolve the vulnerability.
+3. The project creates a new release of the package the vulnerability affects to deliver its fix.
+4. The project publicly announces the vulnerability and describes how to apply the fix.
+
+## Scala Steward
+
+We strongly recommend users of our libraries to use [Scala
+Steward](https://github.com/scala-steward-org/scala-steward) or something
+similar to automatically receive updates.
+
+## Scala Center Security Team
```

Comment:
This is optional, if we have an email up above, we can probably just leave this out. I only have this in here since I've seen others have this listed.

Reply:
I don’t think we have much to say for now.

Reply:
Sounds good I'll remove the heading then.